engine/class_modules/apl/mage.cpp

```diff
@@ -361,18 +361,18 @@ void frost( player_t* p )
   aoe_ff->add_action( "frostbolt" );
   aoe_ff->add_action( "call_action_list,name=movement" );
 
-  aoe_ss->add_action( "cone_of_cold,if=talent.coldest_snap&cooldown.frozen_orb.remains&(prev_gcd.1.comet_storm|prev_gcd.1.frozen_orb&cooldown.comet_storm.remains)&(!talent.deaths_chill|buff.icy_veins.remains<8|buff.deaths_chill.stack>=12)" );
+  aoe_ss->add_action( "cone_of_cold,if=talent.coldest_snap&!action.frozen_orb.cooldown_react&(prev_gcd.1.comet_storm|prev_gcd.1.frozen_orb&cooldown.comet_storm.remains>5)&(!talent.deaths_chill|buff.icy_veins.remains<8|buff.deaths_chill.stack>=12)", "Hold CoC if CMS is coming up in the next 5 seconds + superhuman orb reaction fixed" );
```

Comment:
APL comments, if present, should explain what the line is doing, not what the changes compared to some previous version (that users won't have access to anyways) are.

Reply:
Other APL comments have the same issue.